frontend/src/metabase/ui/components/inputs/NumberInput/NumberInput.tsx

```diff
@@ -5,10 +5,10 @@ import type {
 import type { ChangeEvent, FocusEvent, Ref } from "react";
 import { forwardRef, useLayoutEffect, useState } from "react";
 
-import type { NumberValue } from "metabase/querying/filters/hooks/use-number-filter";
-
 import { TextInput } from "../TextInput";
 
+type NumberValue = number | "";
```

Comment:
Use the same type as beforehttps://github.com/metabase/metabase/blob/2df4df7a8c1b43504109c668fb9d9ac6c6c92488/frontend/src/metabase/querying/filters/hooks/use-number-filter/types.ts#L10